corehq/apps/app_manager/tasks.py

```diff
@@ -67,9 +67,9 @@ def prune_auto_generated_builds(domain, app_id):
 
 
 @task(serializer='pickle', queue='background_queue', ignore_result=True)
-def update_linked_app_and_notify_task(domain, app_id, user_id, email):
+def update_linked_app_and_notify_task(domain, app_id, master_app_id, user_id, email):
```

Comment:
This would break any linked app tasks that are in the queue during deploy since they wouldn't provide the `master_app_id`. There's probably few enough of these calls for that to be fine, but wanted to flag in case its unintentional

Reply:
Ugh, you're right. I'm not planning to merge this PR as-is, but I added a note to the rollout plan to handle the param change - https://docs.google.com/a/dimagi.com/document/d/1dXiVcm871UQIqi5-tFD7rTcLemBxxikpaCiFr6gmkLI/edit?disco=AAAADO5txdY